meinberlin/assets/scss/components/_header.scss

```diff
@@ -187,10 +190,14 @@
     }
 }
 
+// compensate for generic margin-bottom on header and move those elements
+// underneath the "dialograhmen"
+$header-overlap: -2.5em;
```

Comment:
This value is predetermined by the dialograhmen image, so it would be better to define it in `px`. We could either use `calc` to combine it with `$messages-margin-bottom` or define `$messages-margin-bottom` in `px` as well. Or not. I guess this is not too important.